providers/flagd/e2e/evaluation_test.go

```diff
@@ -0,0 +1,40 @@
+//go:build e2e
+// +build e2e
```

Comment:
minor - we don't need both and only keep `//go:build e2e`. This is because of the minimum Go version support which is 1.18+

Reply:
Ya my editor (or the go compiler?) automatically adds line 2 when I add line 1. I'm not sure if it's just a VS code thing or not.